lib/cylc/param_expand.py

```diff
@@ -87,7 +87,7 @@ def item_in_list(item, lst):
         int(item)
     except ValueError:
         return False
-    return int(item) in [int(i) for i in lst]
+    return int(item) in (int(i) for i in lst)
```

Comment:
Can we use `seq` (for sequence) instead of `lst`? I was reading *first* instead of *list* here!

Reply:
>  I was reading _first_ instead of _list_ here

?

Reply:
`lst` and `1st`?